charts/openmetadata/templates/service.yaml

```diff
@@ -3,11 +3,27 @@ kind: Service
 metadata:
   name: {{ include "OpenMetadata.fullname" . }}
   labels:
-    {{- include "OpenMetadata.labels" . | nindent 4 }}
-  {{- with .Values.service.annotations }}
+    app.kubernetes.io/name: '{{ include "OpenMetadata.name" .}}'
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: "{{ .Release.Service }}"
+    app.kubernetes.io/instance: "{{ .Release.Name }}"
+    app.kubernetes.io/component: "openmetadata"
+    {{- if .Values.labels }}
+    {{- range $key, $val := fromYaml .Values.labels }}
+    {{ $key }}: {{ $val | quote }}
+    {{- end}}
+    {{- end}}
+
   annotations:
+    {{- with .Values.service.annotations }}
```

Comment:
We should only be adding `.Values.service.annotations` if they are available!

Reply:
annotations will be only added from .Values.service.annotations if they are available